src/widgets/allergies/allergy-form.component.tsx

```diff
@@ -495,7 +486,7 @@ export default function AllergyForm(props: AllergyFormProps) {
           background: "var(--omrs-color-bg-medium-contrast)"
         }}
       >
-        {patientAllergy && allergyReaction && (
+        {!!(patientAllergy && allergicReactions?.length) && (
```

Comment:
Boolean cast happens implicitly with `&&`. And anyway we should always prefer `Boolean(isWhatever)` over `!!isWhatever`, unless we're playing code golf.